instagram/src/components/CommentSection/Comment.js

```diff
@@ -0,0 +1,22 @@
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Comment.css';
+
+const Comment = props => {
+  return (
+    <div className="comment-text">
+      <span className="comment">{props.comment.text}</span>{' '}
+      <span className="user">-{props.comment.username}</span>
+    </div>
+  );
+};
+
+Comment.propTypes = {
```

Comment:
Good use of prop types here. I encourage you to use prop types wherever possible.